integration/integration_test.py

```diff
@@ -463,6 +463,46 @@ def generate_column(self, size, name=None):
         return self.column_class(name, size, is_valid, values)
 
 
+class FixedSizeBinaryType(PrimitiveType):
+
+    def __init__(self, name, byte_width, nullable=True):
+        PrimitiveType.__init__(self, name, nullable)
```

Comment:
minor: I think `super(..).__init__(..)` is a little better

Reply:
fixed